packages/pg-gateway/biome.json

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+	"files": {
+		"include": ["src/**.ts", "test/**.ts"]
+	},
+	"organizeImports": {
+		"enabled": true
+	},
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true
+		}
+	},
+	"formatter": {
+		"enabled": true
+	}
```

Comment:
Could we keep consistent formatting with other Supabase projects (2 space indents, single quotes)?

Reply:
Done in https://github.com/supabase-community/pg-gateway/pull/1/commits/0a7cb3c44f2a3dd724535a9a70ff483d88e24d8b

Reply:
Let's apply the formatting to the examples as well, looks like those missed the update